tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
I'm not sure I really see the case where you're not in a git repo AND running tools/toc.sh